src/ResizeOptions.php

```diff
@@ -31,6 +31,11 @@ class ResizeOptions implements ResizeOptionsInterface
      */
     private $bypassCache = false;
 
+    /**
+     * @var bool
+     */
+    private $forceReEncoding = false;
```

Comment:
Why „encoding“? What is it encoding? It forces to render the image even if dimensions match to consider exif data right?

Reply:
It always creates a generated image *(and does to-RGB-conversion, interlacing, metadata-stripping, sets sampling factors and JPEG-quality)*, even if dimensions match and no EXIF-orientation is present.

Reply:
Where and why do we need that? I just wonder if we really need a flag (and why) or if we can be smart about it :)

Reply:
Where and why do we need that? I just wonder if we really need a flag (and why) or if we can be smart about it :)

Reply:
If you have for example CMYK images or 100%-quality (and thus very large) images, you never want to serve these images to the user.

Reply:
So internally you call it „processing an image“. Why is the flag called „forceReEncoding“ then? Also, why re- anyway? 😄 What about „forceProcessing“ then?

Reply:
Agree, `forceProcessing` and `contao.image.force_processing` sounds better :)

@contao/developers WDYT?

Reply:
@Toflar and I just discussed this and we think it is probably better to invert the flag and call it `skipIfDimensionsMatch` and `contao.image.skip_if_dimensions_match`

Reply:
I agree.

Reply:
7959c83cc18bd3b5d761ec185ab3ccb07380dc80